packages/console/src/pages/SignInExperience/utilities.ts

```diff
@@ -84,3 +85,44 @@ const emptyTranslation = (translation: Translation): Translation =>
   }, {});
 
 export const createEmptyUiTranslation = () => emptyTranslation(en.translation);
+
+export const getBrandingErrorCount = (
+  errors: FieldErrorsImpl<DeepRequired<SignInExperienceForm>>
+) => {
+  const { color, branding } = errors;
+  const colorFormErrors = color ? Object.keys(color).length : 0;
+  const brandingFormErrors = branding ? Object.keys(branding).length : 0;
+
+  return colorFormErrors + brandingFormErrors;
+};
+
+export const getSignUpAndSignInErrorCount = (
+  errors: FieldErrorsImpl<DeepRequired<SignInExperienceForm>>
+) => {
+  const { signUp, signIn } = errors;
+
+  const signUpErrorMessage = conditional(signUp?.identifier && signUp.identifier.message);
+  const signUpErrors = signUpErrorMessage
+    ? signUpErrorMessage === SignUpIdentifier.EmailOrSms
```

Comment:
It is confusing why signUpErrorMessage value could equals to a SignUpIdentifier?

Reply:
Yeah, refactored and checked with the form data.